earthpy/clip.py

```diff
@@ -2,6 +2,7 @@
 
 "A module to clip vector data using geopandas"
 
+# TODO: Clip poly should use OVERLAY not spatial indexing + intersects
```

Comment:
@lwasser should this be an issue on the GitHub repo instead of a comment?

Reply:
it should but also it's how i personally track code i'm working on! it's nice and bright green in pycharm !! yes TODO is a thing! So i actually want to keep it here as well. issues in github are super useful but i appreciate these types of comments in the code too.